### PR TITLE
Fix rewards claim button for aggregate challenges

### PR DESCRIPTION
**Pull request by dharit-tan**
### Description
Fixes bug where rewards claim button was COMMON instead of PRIMARY_ALT for aggregate challenges.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

Local web

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by rickyrombo — APPROVED** [review]

**Review by piazzatron — APPROVED** [review]

**Comment by dharit-tan** [comment]
Ug realized that this component was used elsewhere. Solution is bit ugly but works. Considered splitting into its own component but decided against it. Lmk if suggestions.

**Review by sddioulde — APPROVED** [review]

**Review by rickyrombo — CHANGES_REQUESTED** [review]
Thinking more on this, we should probably not couple the challenges specific code to the `ButtonWithArrow` component. Let's keep `ButtonWithArrow` generic.

Instead of this, let's pass the buttonType in at the call sites

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rt-chal-button

**Review by rickyrombo — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rt-chal-button

File: packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx [hunk 1]
```diff
@@ -139,15 +145,23 @@ const RewardPanel = ({
...
+      {hasDisbursed ? (
+        <Button
+          className={wm(cn(styles.panelButton, styles.completed))}
+          type={buttonType}
+          text={buttonMessage}
+          onClick={openRewardModal}
+          textClassName={styles.panelButtonText}
+        />
+      ) : (
+        <ButtonWithArrow
```
Comment (rickyrombo) [inline packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx:157]: at this point, maybe we just axe the `ButtonWithArrow` usage entirely and put the icon in above conditionally on the `hasDisbursed` flag? See convo in slack also

**Review by rickyrombo — APPROVED** [review]
Good with merging as is so we can cherry pick if needed, but I think I'm actually leaning on getting rid of the `ButtonWithArrow` component now (sorry for all the back and forth!)

**Comment by dharit-tan** [comment]
> Good with merging as is so we can cherry pick if needed, but I think I'm actually leaning on getting rid of the `ButtonWithArrow` component now (sorry for all the back and forth!)

Agree! I'll merge this for now and follow up with a different PR.

**Pull request merged**